ci/env/kava-internal-testnet/genesis.json

```diff
@@ -618,6 +622,10 @@
             {
               "denom": "xrpb",
               "amount": "103000000000"
+            },
+            {
+              "denom": "erc20/bitgo/wbtc",
```

Comment:
coins must be sorted alphabetically by denom

you can validate the genesis locally with
```
make install
kava validate-genesis ci/env/kava-internal-testnet/genesis.json
```

Reply:
^Thanks for the feedback. @pirtleshell Do we no longer have the "evmutils" module in the genesis.json? We used to need the contract addresses add there as well (our frontend confluence documentation around this still references it).